dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -22,8 +22,8 @@ cdi12.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for
 cdi=Contexts and Dependency Injection
 cdi.desc=Defines the behavior of the Contexts and Dependency Injection (CDI).
 
-cdi.enableImplicitBeanArchives.name=Enable scanning implicit bean archives. This property applies to CDI 1.2 or newer.
-cdi.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries. The default value is true.
+cdi.enableImplicitBeanArchives.name=Enable scanning implicit bean archives.
+cdi.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries. The default value is true. Only applies to CDI 1.2 or newer.
 
-cdi.emptyBeansXMLExplicitBeanArchive.name=An archive with an empty beans.xml file is an explicit bean archive. This property applies to CDI 4.0 or newer.
-cdi.emptyBeansXMLExplicitBeanArchive.desc=CDI 4.0 and later considers an archive with an empty beans.xml file to be an implicit bean archive. This property maintains the existing behavior before to CDI 4.0. The default value is false.
+cdi.emptyBeansXmlCDI3Compatibility.name=CDI 3.0 compatibility for empty beans.xml files.
+cdi.emptyBeansXmlCDI3Compatibility.desc=CDI 4.0 and later consider an archive with an empty beans.xml file to be an implicit bean archive (bean-discovery-mode=ANNOTATED). This property allows an empty beans.xml file to be treated as it was in CDI 3.0 (bean-discovery-mode=ALL). The default value is false. Only applies to CDI 4.0 or newer.
```

Comment:
https://github.com/orgs/OpenLiberty/teams/message-reviewer As part of changing the attribute name we also changed these NLS properties because we realised that the main use of "name" was actually in the WDT tooling and should be rather shorter than it was before.